packages/protocol-kit/src/Safe.ts

```diff
@@ -114,10 +120,38 @@ class Safe {
   async #initializeProtocolKit(config: SafeConfig) {
     const { provider, signer, isL1SafeSingleton, contractNetworks } = config
 
-    this.#safeProvider = new SafeProvider({
-      provider,
-      signer
-    })
+    const isPasskeySigner = signer && typeof signer !== 'string'
```

Comment:
Disclaimer: The comment has nothing  to do with this line but wasnt the protocol kit supposed to have a `getPasskeyCreationOptions` or something like that for the people to have a schema on how to create a "safe-compliant" passkey or something?